internal/lsp/completions/providers/builtins_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"strings"
 	"testing"
 
+	"golang.org/x/net/context"
```

Comment:
This shouldn't be used anymore, probably an auto import

Reply:
oops, thanks, have added a new commit to address